python/tvm/autotvm/env.py

```diff
@@ -27,12 +27,18 @@ def __init__(self):
         self.in_tuning = False
         self.silent = False
 
+    def deepcopy(self, global_scope):
+        self._old = AutotvmGlobalScope.current
+        AutotvmGlobalScope.current = global_scope
```

Comment:
```suggestion
```
This is not necessary as `AutotvmGlobalScope.current == GLOBAL_SCOPE` should always hold

Reply:
ok